packages/third-parties/src/google/ga.tsx

```diff
@@ -0,0 +1,55 @@
+'use client'
+// TODO: Evaluate import 'client only'
+import React, { useEffect } from 'react'
```

Comment:
Just like Josh's feedback in your other PR, we probably shouldn't have to import `React` here

Reply:
Yes I believe so! Keeping the import for now. Will follow up with another PR to remove it, if possible

Reply:
It should be able to remove later, as the tsconfig.json jsx transform is updated to "react-jsx". The change is just landed on canary.